site/en/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,1356 @@
+        "# `prefecth` lets the dataset fetches batches, asynchronously while the model is training.\n",
+        "ds = ds.prefetch(1)\n",
+        "ds"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "GBBZMSuAmQVL",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Pipe the dataset to a model\n",
+        "\n",
+        "Fetch a copy of MobileNet v2 from `tf.keras.applications`.\n",
+        "\n",
+        "This will be used for a simple transfer learning example.\n",
+        "\n",
+        "Set the MobileNet weights to be non-trainable:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "KbJrXn9omO_g",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "mobile_net = tf.keras.applications.MobileNetV2(input_shape=(192, 192, 3), include_top=False)\n",
+        "mobile_net.trainable=False"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Y7NVWiLF3Vbf",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "This model expects it's input to be normalized to the `[-1,1]` range:\n",
+        "\n",
+        "```\n",
+        "help(keras_applications.mobilenet_v2.preprocess_input)\n",
+        "```\n",
+        "\n",
+        "<pre>\n",
+        "...\n",
+        "This function applies the \"Inception\" preprocessing which converts\n",
+        "the RGB values from [0, 255] to [-1, 1] \n",
+        "...\n",
+        "</pre>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "CboYya6LmdQI",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "So before the passing it to the mobilnet model, we need to convert the input from a range of `[0,1]` to `[-1,1]`."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "SNOkHUGv3FYq",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "def change_range(image,label):\n",
+        "  image = 2*image-1\n",
+        "  return image, label\n",
+        "\n",
+        "keras_ds = ds.map(change_range)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "QDzZ3Nye5Rpv",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The mobilenet returns a `6x6` spatial grid of features for each image.\n",
+        "\n",
+        "Pass it a batch of images to see:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "OzAhGkEK6WuE",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# The dataset may take a few seconds to start, as it fills its shuffle buffer.\n",
+        "image_batch, label_batch = next(iter(keras_ds))"
```

Comment:
@jsimsa I forget: did we discuss adding a `Dataset.first()` method to make this a bit cleaner? (IIRC it was going to make some of the reducer examples a lot nicer.)